_posts/2020-10-19-Software-Synth.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Software Synth in Jamulus"
+lang: "en"
+author: "niebert"
+heading: "Software Synth in Jamulus"
+---
+This post will discuss how to setup a software synth in Jamulus.
+<!--more-->
+
+## Linux
+We will elaborate the concept of playing a software synth on Linux as example and transfer that to other operating systems (e.g. Windows, MacOSX).
```

Comment:
* While "elaborate" isn't 100% wrong, it's not as usual as "develop" is, hence "develop" should probably be preferred here.
* missing word: "as example" -> "as an example"